plugin-gradle/README.md

```diff
@@ -260,7 +261,7 @@ configure<com.diffplug.gradle.spotless.SpotlessExtension> {
 [homepage](https://github.com/pinterest/ktlint). [changelog](https://github.com/pinterest/ktlint/releases).  Spotless does not ([yet](https://github.com/diffplug/spotless/issues/142)) respect the `.editorconfig` settings ([ktlint docs](https://github.com/pinterest/ktlint#editorconfig)), but you can provide them manually as `userData`.
 
```

Comment:
As we're updating README.md, let's update line 84: point the link to https://plugins.gradle.org/plugin/com.diffplug.spotless (I believe spotless migrated to this one)

Reply:
Ah, good call, thanks!